TODO/data-flow-in-vue-and-vuex.md

```diff
@@ -1,32 +1,32 @@
 > * 原文地址：[DATA FLOW IN VUE AND VUEX](https://benjaminlistwon.com/blog/data-flow-in-vue-and-vuex/)
 * 原文作者：[Benjamin Listwon](https://benjaminlistwon.com/about/)
 * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者：
+* 译者：[linpu.li](https://github.com/llp0574)
 * 校对者：
 
-It seems like one of the things that trips people up in [Vue](https://vuejs.org) is how to share state across components. For those new to reactive programming, something like [vuex](https://github.com/vuejs/vuex/) can seem daunting with loads of new jargon and the way it separates concerns. It can also seem like overkill when all you want is to share one or two pieces of data.
+看起来在 [Vue](https://vuejs.org) 里面困扰开发者的事情之一是如何在组件之间共享状态。对于刚刚接触响应式编程的开发者来说，学习 [Vuex](https://github.com/vuejs/vuex/) 这种库，伴随着许多新名词及其分离关注点的方式，似乎是件非常难的事情。
```

Comment:
> ... something like vuex can seem daunting with loads of new jargon and the way it separates concerns. It can also seem like overkill when all you want is to share one or two pieces of data.

感觉没有译出原来的味道，可以改为：**繁多的新名词和关注点分离的方式令人望而生畏**。同时漏了一句：特别是当你只希望分享一两个数据片段时，（这一套逻辑的复杂性）就显得有点过分了。